website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -39,6 +39,57 @@ resource "aws_autoscaling_group" "bar" {
 }
 ```
 
+### Create predictive scaling policy using custom metrics
+```terraform
+resource "aws_autoscaling_policy" "example" {
+  autoscaling_group_name = "my-test-asg"
+  name                   = "foo"
+  policy_type            = "PredictiveScaling"
+  predictive_scaling_configuration {
+    metric_specification {
+      target_value = 10
+      customized_load_metric_specification {
+        metric_data_queries {
+          id         = "load_sum"
+          expression = "SUM(SEARCH('{AWS/EC2,AutoScalingGroupName} MetricName=\"CPUUtilization\" my-test-asg', 'Sum', 3600))"
+        }
+      }
+      customized_scaling_metric_specification {
```

Comment:
Is the use-case valid? You would define a policy with `customized_scaling_metric_specification` and `customized_capacity_metric_specification` defined?

Reply:
Yes, the `customized_capacity_metric_specification` is optional and can only be used when the load metric is `customized_load_metric_specification `, it has nothing to do with scaling metrics. 

Some more context, the example I gave was translated from AWS blog that show customer how to use custom metric specification to solve Blue/Green deployment problem: https://aws.amazon.com/blogs/compute/retaining-metrics-across-blue-green-deployment-for-predictive-scaling/